hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -28,7 +28,8 @@
                         "^UVM_WARNING\\s[^:].*$",
                         "^Assert failed: ",
                         "^\\s*Offending '.*'",
-                        "^TEST FAILED (UVM_)?CHECKS$"]
+                        "^TEST FAILED (UVM_)?CHECKS$",
+                        "^Error:.*$]  // ISS errors
```

Comment:
Did you miss a back quotation here `"^Error:.*$"]`

Reply:
Oh my... Fixed.